### ISSUE TITLE
Docs don't show up for imported functions in autocomplete

### ISSUE THREAD

**Issue by bengolds** [elm-tooling/elm-language-server#376]
When using autocomplete, docs show up for locally defined functions/types and imported types. However, imported functions don't show docs. This is pretty important, since most of the docs one usually needs are for imported functions!

Using the sample file below:
```
module Main exposing (..)

import Foo
import Html exposing (..)


type alias MyList =
    List Int


myFunction : String -> List String
myFunction =
    String.split ","


main =
    Html.text ""

```

I get the following behaviors when using autocomplete:

## Locally defined type
<img width="353" alt="Screenshot 2020-11-25 at 09 08 30@2x" src="https://user-images.githubusercontent.com/446007/100260042-cf08dc00-2efd-11eb-8402-0d354a7f6579.png">

## Locally defined function
<img width="530" alt="Screenshot 2020-11-25 at 09 07 58@2x" src="https://user-images.githubusercontent.com/446007/100259977-bac4df00-2efd-11eb-8b2b-e612b0bfceca.png">

## Imported type
<img width="907" alt="Screenshot 2020-11-25 at 09 08 58@2x" src="https://user-images.githubusercontent.com/446007/100260091-df20bb80-2efd-11eb-9b4b-1d04ae1b8faa.png">

## Imported function
<img width="520" alt="Screenshot 2020-11-25 at 09 09 25@2x" src="https://user-images.githubusercontent.com/446007/100260158-f1025e80-2efd-11eb-9cc1-c67f3d9851f4.png">

I'm on macOS Big Sur; this repros on NeoVim and VSCode.

**Comment by Razzeee** [comment]
Will be fixed by [`471a32a` (#378)](https://github.com/elm-tooling/elm-language-server/pull/378/commits/471a32aab4564edf29b5c7c2278869a60942e7c5)

might be worth a backport, as the next big version is probably a bit more of

**Comment by Razzeee** [comment]
Closing as this has been fixed and will be in the next release.
